src/Augurk.Api/Controllers/V2/FeatureV2Controller.cs

```diff
@@ -98,15 +107,29 @@ public class FeatureV2Controller : ApiController
 
             var response = Request.CreateResponse(HttpStatusCode.Created);
 
+            DbFeature dbFeature = null;
             try
             {
-                await _featureManager.InsertOrUpdateFeatureAsync(feature, productName, groupName, version);
+                dbFeature = await _featureManager.InsertOrUpdateFeatureAsync(feature, productName, groupName, version);
             }
             catch (Exception exception)
             {
                 response = Request.CreateErrorResponse(HttpStatusCode.InternalServerError, exception);
             }
 
+            // Run the analysis for this feature
+            if (dbFeature != null)
+            {
+                try
+                {
+                    await _analyzer.AnalyzeAndPeristResultsAsync(productName, version, dbFeature);
+                }
+                catch
+                {
+                    // Don't do anything, a broken analysis is no reason to report an error
```

Comment:
Are we in agreement about this? I think it is reasonable, at the very least until we have finished the entire feature in master....